shell/platform/fuchsia/flutter/external_view_embedder.cc

```diff
@@ -108,13 +108,16 @@ flutter::PostPrerollResult ExternalViewEmbedder::PostPrerollAction(
 }
 
 void ExternalViewEmbedder::BeginFrame(
-    SkISize frame_size,
     GrDirectContext* context,
-    double device_pixel_ratio,
     const fml::RefPtr<fml::RasterThreadMerger>& raster_thread_merger) {
   TRACE_EVENT0("flutter", "ExternalViewEmbedder::BeginFrame");
+}
```

Comment:
Remove the trace event - it's pure overhead at this point, measuring an empty function.